openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -1287,19 +1287,19 @@ def layer_structure_print(info: dict) -> None:
                                 )
                                 tf_layers_dict[layer_id] = \
                                     tf.maximum(0.0, inp) + \
-                                        tf.minimum(0.0, temp_alpha * inp)
+                                        tf.minimum(0.0, temp_alpha.transpose(1, 2, 0) * inp)
```

Comment:
Aborts when `alpha_len` is 2. Therefore, I think we need to process branch `elif alpha_len >= 2:`.

Reply:
added abort condition when ```alpha_len``` is not 4, 3, or 1.